modules/apps/powwow/powwow-portlet/docroot/meetings/view_meeting.jsp

```diff
@@ -45,12 +46,14 @@ portletURL.setParameter("backURL", backURL);
 
 	<%@ include file="/meetings/meeting_body.jspf" %>
 
+	<br>
```

Comment:
even using `<br>` is an acceptable way, are you able to adjust using CSS styling instead?